lib/trento/domain/cluster/cluster.ex

```diff
@@ -241,6 +243,20 @@ defmodule Trento.Domain.Cluster do
     |> Multi.execute(&maybe_emit_cluster_health_changed_event/1)
   end
 
+  def execute(
+        %Cluster{
+          cluster_id: cluster_id
+        } = cluster,
+        %CompleteChecksExecutionWanda{
+          cluster_id: cluster_id
+        } = command
+      ) do
+    cluster
+    |> Multi.new()
+    |> Multi.execute(&maybe_emit_cluster_checks_health_changed_event(&1, command))
```

Comment:
I was initially thinking _would emitting a `ClusterHealthChanged` be enough?_ 

Probably not, I see that this new event gives a higher granularity of what is happening in the system and it becomes interesting when we want to react upon change of the aggregated result of a check execution.

So far only the UI would be interested in reacting to such an event, right?

Reply:
At the end, we need to change the aggregate with this new checks health value to compute the final value, so an event must exist for that (we cannot change the aggregate otherwise). 
So yes, it is simply an intermediate event (the same way we had the same event for the discovered values from the agent).

Maybe there is some better way to handle everything only with the `ClusterHealthChanged`, but at the end, this event would become more complicated, as it would need to take care of changing the aggregate in different scenarios and with different events.